src/avram/query_builder.cr

```diff
@@ -322,6 +328,9 @@ class Avram::QueryBuilder
         [clause, sql_clause.conjunction.to_s]
       end
 
+      # Remove blank conjunctions
+      statements.reject!(&.blank?)
```

Comment:
By adding in the `Avram::Where::Conjunction::None`, the `statements` might look like `["(", "", "users.age = $1", "AND", ")", "AND"]`